### PR TITLE
Comments (model.js, [server.js]) => Fixed comments issue

### PR DESCRIPTION
**Pull request by iambenkay**
I just fixed some comments. I still think securing the API should be a thing. Don't you think so?

### CODE REVIEW

File: src/validators.js [hunk 1]
```diff
@@ -51,6 +51,7 @@ const extractParams = (req, res, next) => {
 		if (!req['recordId'] && pathParams[2]) {
 			req['recordId'] = isHexString.test(pathParams[2]) ? pathParams[2] : undefined;
 		}
+		console.log(req.headers)
```
Comment (vasanthv) [inline src/validators.js]: Can we remove this log?

Comment (iambenkay) [reply src/validators.js]: Yes! Sorry about that I was testing and forgot it in there

**Comment by iambenkay** [comment]
Okay I have made the changes. Sorry again about that.

File: src/validators.js [hunk 2]
```diff
@@ -10,14 +10,14 @@ const removeNativeKeys = (req, res, next) => {
...
+// validator: size of payload < 100KB
 const sizeValidator = (req, res, next) => {
 	if (req.method === 'POST' || req.method === 'PUT') {
 		if (Object.keys(req.body).length > 0) {
 			const memorySize = helper.memorySizeOf(req.body);
 			req['bodySize'] = memorySize;
-			// memorySize is size in bytes
-			if (memorySize > 100000) {
+			// memorySize is size in bytes => 1024 * 1024
+			if (memorySize > (1024 * 1024)) {
```
Comment (vasanthv) [inline src/validators.js]: @iambenkay sorry I didn't note this while reviewing it first. 100KB = 100 * 1024

**Comment by vasanthv** [comment]
Fixed the changes myself. Thanks for the changes :)

**Pull request merged**